src/DistortableCollection.js

```diff
@@ -100,6 +101,15 @@ L.DistortableCollection = L.FeatureGroup.extend({
       /* conditional prevents disabled images from flickering multi-select mode */
       if (layer.editing.enabled()) {
         L.DomUtil.toggleClass(e.target, 'collected');
+        // re-order layers by ... to match their display order in UI
```

Comment:
Ah, sorry, i meant you could fill in '...' with what it's ordering by -- which i believe would be `layer._leaflet_id`?